src/section/section_members.rs

```diff
@@ -182,30 +181,26 @@ impl SectionMembers {
         }
     }
 
-    /// Remove a member from our section. Returns the SocketAddr and the state of the member before
-    /// the removal.
-    pub fn remove(&mut self, name: &XorName) -> (Option<SocketAddr>, MemberState) {
+    /// Remove a member from our section. Returns the `MemberInfo` from before the removal.
```

Comment:
what is `from before the removal` ?

Reply:
Means it returns the state as it were before the member was removed. But that's probably expected anyway and this commented is just redundant and confusing. I'll remove it.